apps/spark/src/main/java/com/linkedin/openhouse/jobs/util/TableStatsCollectorUtil.java

```diff
@@ -236,4 +270,54 @@ private static long getSumOfFileSizeBytes(Dataset<Row> allDataFiles) {
         .first()
         .getLong(0);
   }
+
+  /**
+   * Get sharingEnabled from table policies.
+   *
+   * @param table
+   * @return
+   */
+  private static Boolean getTablePoliciesSharingEnabled(Table table) {
```

Comment:
table policies are stored as a JSON string under `table.properties()`, so using JSON parsing to get the values inside the nested string